src/static/skins/colibris/pad.js

```diff
@@ -1,4 +1,32 @@
 function customStart()
 {
   $('#pad_title').show();
+
+  var showMoreIconBtn = $('<span class="show-more-icon-btn"></span>')
+  showMoreIconBtn.click(function() {
+    $('.toolbar').toggleClass('full-icons');
+    $('#editorcontainer').css('top', $('.menu_left').height() + 1 + 'px');
+  })
+  $('.toolbar').append(showMoreIconBtn)
+
+  var timer;
+  // on resize end
+  window.onresize = function(){
```

Comment:
As said in the discussion, hooking on a window resize event is never a good idea. @seballot already explained why he had to do this, and @JohnMcLear had to do the same in 2014 in commit 0685e563ed94, for similar reasons.